sample/buildSrc/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ repositories {
 }
 
 dependencies {
-    implementation("com.dropbox.affectedmoduledetector:affectedmoduledetector:0.1.0-SNAPSHOT")
+    implementation("com.dropbox.affectedmoduledetector:affectedmoduledetector:0.1.0")
```

Comment:
This was intentionally set to `-SNAPSHOT` so that developers could test their changes by pushing to their local maven repo.  

I probably also need to update the release notes to ensure this value is bumped, otherwise the sample will use an out of date version of the library.